Modelica/Mechanics/Translational/Components/Brake.mo

```diff
@@ -1,75 +1,85 @@
 within Modelica.Mechanics.Translational.Components;
 model Brake "Brake based on Coulomb friction"
-
   extends Modelica.Mechanics.Translational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
   extends Modelica.Thermal.HeatTransfer.Interfaces.PartialElementaryConditionalHeatPortWithoutT;
+
+  import Modelica.Blocks.Types.{ExternalCombiTable1D, Extrapolation, Smoothness};
+  import Modelica.Blocks.Tables.Internal.{getTable1DValue, getTable1DValueNoDer, getTable1DValueNoDer2};
+
   parameter Real mu_pos[:, 2]=[0, 0.5]
-    "Positive sliding friction coefficient [-] as function of v_rel [m/s] (v_rel>=0)";
+    "Positive sliding friction coefficient [-] as function of v [m/s] (v>=0)";
```

Comment:
Why is this description string oscillating back and forth?
In MSL 3.2.3 it was `v` - and this indicates that it was changed to `v_rel` and is now changed back to `v`.

Reply:
Actually, the model itself contains no such variable. Either `v` or `v_relfric`. There is used `v` in the model's documentation, so this shall be the same for consistency.

Reply:
See also #3694.